src/components/SendModalComponent.vue

```diff
@@ -42,6 +45,21 @@ onMounted(async () => {
 });
 
 async function sendAsset() {
+  // Check network and update previous values
+  let chainID = await web3.eth.getChainId();
+  if (chainID === 22) {
+    isL14.value = true;
+    isL16.value = false;
+  } else if (chainID === 2828) {
+    isL16.value = true;
+    isL14.value = false;
+  } else {
+    wrongNetwork.value = true;
+    isL14.value = false;
+    isL16.value = false;
+    return;
+  }
```

Comment:
might be better to use a "switch / case" here instead of if/else as in the else if you are checking for "chainId" again.